src/VerifiableFactory.sol

```diff
@@ -9,8 +9,6 @@ interface IProxy {
     function salt() external view returns (uint256);
 
     function owner() external view returns (address);
-
-    function creator() external view returns (address);
```

Comment:
It looks like you deleted it from the interface but not from the code?

Thinking about it, it may still be useful to have; otherwise there's no way to know which factory to check if all you have is an instance.

Reply:
the interface here is only under the `VerifiableFactory` scope, the actual `creator` is a public immutable variable which will have the auto-generated getter interface, do we need to have it explicitly under VerifiableFactory if not in use?

Reply:
Ah, good point. No, but we should be having the proxy implement the same interface we use here - otherwise there could be a mismatch between what the proxy implements and the interface specifies and we wouldn't know at compile-time.